bigquery_etl/dryrun.py

```diff
@@ -113,7 +113,7 @@
     "sql/moz-fx-data-shared-prod/mozilla_vpn_derived/protected_v1/init.sql",
     "sql/moz-fx-data-shared-prod/mozilla_vpn_derived/add_device_events_v1/init.sql",
     "sql/moz-fx-data-shared-prod/mozilla_vpn_external/devices_v1/init.sql",
-    "sql/moz-fx-data-shared-prod/search_terms/suggest_impression_sanitized/view.sql",
+    *glob.glob("sql/moz-fx-data-shared-prod/search_terms*/**/*.sql", recursive=True),
```

Comment:
We won't be able to do dry runs on any of these search terms datasets due to the dry run service not having access to the tables we're hitting.